src/k8s/application.yaml

```diff
@@ -0,0 +1 @@
+client.customer-master-data.url: customer-master-data-management-service.customer-master-data.svc.cluster.local:9021
```

Comment:
Hier musst du vermutlich `"customer-master-data-management-service.customer-master-data.svc.cluster.local:9021"` wegen dem `:`.

Reply:
in einem String ist das in einer Zeile egal, ich mach aber mal eins zur Vorsicht hin